backend/apps/menu/serializers.py

```
@@ -0,0 +1,189 @@
+        data['isPublished'] = data.pop('is_published')
+        data['createdAt'] = data.pop('created_at')
+        return data
+
+
+class MenuWeekCreateSerializer(serializers.ModelSerializer):
+
+    week_start = serializers.DateField(required=True)
+    image = serializers.ImageField(required=False, allow_null=True)
+    class Meta:
+        model = MenuWeek
+        fields = ['week_start', 'image']
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)